.github/workflows/jekyll.yml

```diff
@@ -0,0 +1,54 @@
+name: Jekyll site CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.x'
+    - name: Install dependencies
+      run: bundle install
+    - name: Build site and feed
+      run: |
+        # have site.url produce a valid value
+        sed -i -e '/^collections:/i \
+        url: https://rustacean-station.org' \
+               _config.yml
+        bundle exec jekyll build
+    - name: Check dates
+      run: |
+        for date in $(grep -h date _episodes/*.md | sed 's/^date: //'); do
+          date "--date=$date";
```

Comment:
Does this just ensure that the date is parseable by `date`, not that it's RFC 3339?

Reply:
Er, nevermind, I hadn't seen the later commit. :P